src/bot/exts/fun/uwu.py

```diff
@@ -179,18 +40,18 @@ async def uwu_command(self, ctx: Context, *, text: t.Optional[str] = None) -> No
 
         # Grabs the text from the embed for uwuification
         if embeds:
-            embed = messages.convert_embed(self._uwuify, embeds[0])
+            embed = messages.convert_embed(imsosorry.uwuify, embeds[0])
```

Comment:
Handle multiple embeds
```suggestion
            embeds = [messages.convert_embed(imsosorry.uwuify, embed) for embed in embeds]
```